app/models/outbound_call.rb

```diff
@@ -4,8 +4,8 @@
 #
 #  id                :bigint           not null, primary key
 #  call_duration     :string
-#  completed_at      :datetime
 #  from_phone_number :string           not null
+#  note              :text
```

Comment:
I'd love to see a comment in this file (not in the autogenerated part) indicating the lifecycle of a call, so that when we're debugging the `twilio_status` later, we'll know `queued` could mean waiting for user (not client) to pick up.